modules/bibformat/lib/bibreformat.py

```diff
@@ -280,7 +279,7 @@ def outdated_caches(fmt, last_updated, chunk_size=5000):
     max_id = run_sql("SELECT max(id) FROM bibrec")[0][0] or 0
     for start in xrange(1, max_id + 1, chunk_size):
         end = start + chunk_size
-        recids += intbitset(run_sql(sql, (last_updated_str, fmt, start, end)))
```

Comment:
Yes, indeed! Nobody uses `outdated_caches()` as an API... Perhaps we can amend this (and remove unnecessary date statements) in a next commit?